packages/lwc-language-server/src/component-indexer.ts

```diff
@@ -103,7 +112,8 @@ export default class ComponentIndexer extends BaseIndexer {
     }
 
     persistCustomComponents() {
-        const indexPath = join(this.workspaceRoot, CUSTOM_COMPONENT_INDEX_FILE);
+        const indexPath = path.join(this.workspaceRoot, CUSTOM_COMPONENT_INDEX_FILE);
```

Comment:
This was failing on projects that didn't previously had `.sfdx/indexes/lwc/` created.